### PR TITLE
0.3.0 Release to CRAN

### PR DESCRIPTION
**Pull request by elimillera**
### Thank you for your Pull Request! 

We have developed a Pull Request template to aid you and our reviewers.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the xportr codebase remains robust and consistent.  

### The scope of `{xportr}`

`{xportr}`'s scope is to enable R users to write out submission compliant `xpt` files that can be delivered to a Health Authority or to downstream validation software programs.  We see labels, lengths, types, ordering and formats from a dataset specification object (SDTM and ADaM) as being our primary focus. We also see messaging and warnings to users around applying information from the specification file as a primary focus. Please make sure your Pull Request meets this **scope of {xportr}**. If your Pull Request moves beyond this scope, please get in touch with the `{xportr}` team on [slack](https://pharmaverse.slack.com/archives/C030EB2M4GM) or create an issue to discuss.

Please check off each task box as an acknowledgment that you completed the task. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

### Changes Description

_(descriptions of changes)_ 

### Task List

- [x] The spirit of xportr is met in your Pull Request
- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Summary of changes filled out in the above Changes Description.  Can be removed or left blank if changes are minor/self-explanatory.
- [x] Check that your Pull Request is targeting the `devel` branch, Pull Requests to `main` should use the [Release Pull Request Template](https://github.com/atorus-research/xportr/tree/94_pr_template/.github/PULL_REQUEST_TEMPLATE)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/).  Use `styler` package and functions to style files accordingly.
- [x] Updated relevant unit tests or have written new unit tests. See our [Wiki](https://github.com/atorus-research/xportr/wiki/Style-Guide-for-Unit-Tests) for conventions used in this package.
- [x] Creation/updated relevant roxygen headers and examples. See our [Wiki](https://github.com/atorus-research/xportr/wiki/Style-Guide-for-Roxygen-Headers) for conventions used in this package.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Run `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new/updated functions occur on the "Reference" page.
- [x] Update NEWS.md if the changes pertain to a user-facing function (i.e. it has an @export tag) or documentation aimed at users (rather than developers)
- [x] Address any updates needed for vignettes and/or templates
- [x] Link the issue Development Panel so that it closes after successful merging.
- [x] Fix merge conflicts 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: NEWS.md
Comment (averissimo) [inline NEWS.md:8]:
```suggestion
* Fixed bug where character_types were case sensitive. They are now case insensitive (#77).
```

File: NEWS.md
Comment (averissimo) [inline NEWS.md:21]:
```suggestion
* Increase test coverage to 100% (#82)
```

**Review by averissimo — COMMENTED** [review]
Great job!

* Are we keeping the azure configuration?
* The `cran-comments.md` should be updated (or removed from the repo and changed locally before submission)
* RHUB check `rhub::check_for_cran()`
  * I've run it and only got a note complaining that `tidy` command
  * Should be good to go, but @elimillera as the package maintainer should run it

Suggestion for `cran-comments.md`

```
## xportr 0.3.0 

Check Results:
  No Errors or warnings

### Notes:

```
* checking HTML version of manual ... NOTE
Skipping checking HTML validation: no command 'tidy' found
```
  
All words in the description are common acronyms in industry:

  - ADaM - Analysis Dataset Model
  - CDISC - Clinical Data Interchange Standards Consortium
  - SDTM - Standard Data Tabulation Model
  - XPT - SAS Transport File

### Tested on:

- RHub Check Windows (R-devel), Fedora (R-devel), Ubuntu (R-release)
- GitHub R-CMD-Check:
    - Windows R-release
    - MacOS R-release
    - Ubuntu R-oldrel-1
    - Ubuntu R-release
    - Ubuntu R-devel
```

**Comment by elimillera** [comment]
Thanks @averissimo! Rhub check was all green for me other than a 403 we're getting on a link that is expected. I'll push up my cran comments and remove the azure-pipeline

**Comment by bms63** [comment]
Should we put a little blurb underneath helpers to say these are not intended for use, but just for understanding??  Not sure how to phrase it.

Maybe xportr_metadata() should go in the above section as well?

image

**Comment by bms63** [comment]
@elimillera I made a quick PR to address my findings after review of the website.

**Review by vedhav — COMMENTED** [review]
LGTM! I can't wait for the CRAN release.
There were some minor document changes I spotted. Please have a look at PR #168

**Review by cpiraux — APPROVED** [review]

**Review by bms63 — APPROVED** [review]

**Comment by bms63** [comment]
@elimillera  I think we are good to go.

**Review by averissimo — APPROVED** [review]
This was a great collaboration! Great job everyone

**Review by vedhav — APPROVED** [review]

**Comment by elimillera** [comment]
Fantastic work all. Its off to CRAN. I'll post any issues or updates here as we get feedback

**Pull request merged**
